examples/demo.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+<script defer src="../pyscriptjs/build/pyscript.js"></script>
```

Comment:
this surely works locally after a clone but I am wondering if this would work once published live too 🤔